.gitignore

```diff
@@ -4,3 +4,4 @@
 /node_modules
 /package-lock.json
 
+/nbproject/private/
```

Comment:
Please, put this ignore paths on your environment git config file, it does not belong to the project itself.

Reply:
Solved